hooks/tk-multi-publish2/basic/nuke_publish_reviewsubmission.py

```diff
@@ -0,0 +1,333 @@
+        if review_submission_app is None:
+            error_msg = "Item cannot be validated. Review Submission App is not available."
+            self.logger.error(error_msg)
+            return
+        write_node_app = self.parent.engine.apps.get("tk-nuke-writenode")
+        if write_node_app is None:
+            error_msg = "Item cannot be validated. Write Node App is not available."
+            self.logger.error(error_msg)
+            return
+
+        self._send_to_screening_room(item,
+                                     item.parent.properties["sg_publish_data"],
+                                     self.parent.context.task,
+                                     item.description,
+                                     item.get_thumbnail_as_path(),
+                                     lambda *args, **kwargs: None)
+
+    def finalize(self, settings, item):
+        """
+        Execute the finalization pass. This pass executes once all the publish
+        tasks have completed, and can for example be used to version up files.
+
+        :param settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        """
+        pass
+
+    def _send_to_screening_room(self, item, sg_publish, sg_task, comment, thumbnail_path, progress_cb):
+        """
+        Take a write node's published files and run them through the review_submission app 
+        to get a movie and Shotgun Version.
+
+        :param item:            The item to be published
+        :param sg_publish:      The Shotgun publish entity dictionary to link the version with
+        :param sg_task:         The Shotgun task entity dictionary for the publish
+        :param comment:         The publish comment
+        :param thumbnail_path:  The path to a thumbnail for the publish
+        :param progress_cb:     A callback to use to report any progress
+        """
+        write_node_app = self.parent.engine.apps.get("tk-nuke-writenode")
+        review_submission_app = self.parent.engine.apps.get("tk-multi-reviewsubmission")
+
+        render_path = item.parent.properties.get("path")
+        render_template = item.parent.properties.get("work_template")
+        publish_template = item.properties.get("publish_template") or item.parent.properties.get("publish_template")                        
+        render_path_fields = render_template.get_fields(render_path)
+
+        if hasattr(review_submission_app, "render_and_submit_version"):
+            # this is a recent version of the review submission app that contains
+            # the new method that also accepts a colorspace argument.
+            colorspace = item.parent.properties.get("color_space")
+            review_submission_app.render_and_submit_version(
+                publish_template,
```

Comment:
Ok, I see this is required here. I think this should come from the image sequence item, as set by the publish file plugin. That would allow this plugin to work for more than just nuke renders.